3news.co.nz.js

```diff
@@ -1,282 +0,0 @@
-{
-	"translatorID": "a9f7b277-e134-4d1d-ada6-8f7942be71a6",
```

Comment:
I used a different id etc. but was not sure, because I read that it was a merge from the old into something new. Anyway let me know what you think...